docs/storybook/stories/forms/FormFieldCNPJ.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Button, Flex } from '@ttoss/ui';
+import { Form, FormFieldCNPJ, useForm, yup, yupResolver } from '@ttoss/forms';
+import { Meta, Story } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Forms/FormFieldCNPJ',
+  component: FormFieldCNPJ,
+} as Meta;
+
+const schema = yup.object({
+  cnpj: yup.string().required('Value is required'),
```

Comment:
Seria possível implementar uma verificação numérica para o cnpj?
https://www.geradorcnpj.com/javascript-validar-cnpj.htm

Reply:
Referenced in a new issue to implement in the future https://github.com/ttoss/ttoss/issues/513